src/text/text_editor.py

```diff
@@ -252,7 +328,7 @@ async def coroutine() -> None:
 
 def do_about() -> None:
     """About from menu select"""
-    show_message("About", "Text editor demo.\nCreated by Jonathan Slenders.")
+    show_message("About", "Text editor.\nCreated by Adaptable Antelopes.")
```

Comment:
big facts

Reply:
Actually I should prob make it say ThoughtBox isntead of text editor